extensions/omni_credentials/src/instantiate_file_store.sql

```diff
@@ -44,7 +78,38 @@ begin
     insert into credential_file_stores (filename) values (instantiate_file_store.filename);
 
     perform credential_file_store_reload(filename);
-    execute format('copy encrypted_credentials to %L', filename);
+
+    create or replace function update_file(filename text) returns boolean
```

Comment:
Maybe we can find a slightly better name for this function? What do you think about `update_credentials_file`?